### PR TITLE
add Ukrainian translation

### PR DESCRIPTION
**Pull request by ManoylovAC**
Hey Patricio & Jen  👋 
I hope you are doing well.

Thanks for you great book!
I have added a full translation of all sections and a glossary into **Ukrainian** so that everyone can use it. 
I also added a couple of small changes to other files, please review if you don't mind them. These are mostly subtitles and minor edits.

- Added **Ukrainian translation**.
- Subtitles have been added to the files of the appendix section.
- A small fix for saving the language query parameter when clicking the Home button
- Unlocked the "Textures" and "Image Operations" sections.

Sincerely, Andriy Manoylov.

### CODE REVIEW

**Comment by patriciogonzalezvivo** [comment]
Thank you for this great contribution!

File: 13/index.php [hunk 1]
```diff
@@ -26,9 +26,9 @@
 	<ul class="navigationBar" >
 		<li class="navigationBar" onclick="previusPage()">&lt; &lt; Previous</li>
 		<li class="navigationBar" onclick="homePage()"> Home </li>
+		<li class="navigationBar" onclick="nextPage()">Next &gt; &gt;</li>
```
Comment (patriciogonzalezvivo) [inline 13/index.php:29]: The subsequent chapters are not ready. Please comment this

File: README-ua.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    * [Фігури](07/?lan=ua)
+    * [Матриці](08/?lan=ua)
+    * [Патерни](09/?lan=ua)
+
+* Генеративний дизайн
+    * [Випадковість](10/?lan=ua)
+    * [Шум](11/?lan=ua)
+    * [Клітинний шум](12/?lan=ua)
+    * [Фрактальний броунівський рух](13/?lan=ua)
+    * [Фрактали](14/?lan=ua)
```
Comment (patriciogonzalezvivo) [inline README-ua.md:34]: Please don't link chapters 14+

File: README-it.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ Grazie a Nahuel Coppero (Necsoft) per la [traduzione (español)](?lan=es) spagno
 
 Grazie a [Nicolas Barradeau](https://twitter.com/nicoptere) e a [Karim Naaji](http://karim.naaji.fr/) per la [traduzione (français)](?lan=fr) francese
 
-Grazie a [Andrea Rovescalli](https://www.earove.info) per la [traduzione](?lan=it) italiana
+Grazie a [Manoylov Andriy](https://twitter.com/ManoylovAC) per la [traduzione ucraina (українська)](?lan=ua)
```
Comment (patriciogonzalezvivo) [inline README-it.md:97]: Seams Andrea line was deleted here

File: 00/README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ In the following chapters you will discover how incredibly fast and powerful thi
...
+This book will teach you how to use and integrate shaders into your projects, improving their performance and graphical quality. Because GLSL (OpenGL Shading Language) shaders compile and run on a variety of platforms, you will be able to apply what you learn here to any environment that uses OpenGL, OpenGL ES or WebGL. In other words, you will be able to apply and use your knowledge with [Processing](https://processing.org/) sketches, [openFrameworks](http://openframeworks.cc/) applications, [Cinder](http://libcinder.org/) interactive installations, [Three.js](http://threejs.org/) websites or iOS/Android games.
```
Comment (patriciogonzalezvivo) [inline 00/README.md:21]: good catch

File: 11/README.md [hunk 5]
```diff
@@ -202,7 +202,7 @@ All these improvements result in an algorithmic masterpiece known as **Simplex N
 
 Well... enough technicalities, it's time for you to use this resource in your own expressive way:
 
-* Contemplate how each noise implementation looks. Imagine them as a raw material, like a marble rock for a sculptor. What can you say about about the "feeling" that each one has? Squinch your eyes to trigger your imagination, like when you want to find shapes in a cloud. What do you see? What are you reminded of? What do you imagine each noise implementation could be made into? Following your guts and try to make it happen in code.
+* Contemplate how each noise implementation looks. Imagine them as a raw material, like a marble rock for a sculptor. What can you say about the "feeling" that each one has? Squinch your eyes to trigger your imagination, like when you want to find shapes in a cloud. What do you see? What are you reminded of? What do you imagine each noise implementation could be made into? Following your guts and try to make it happen in code.
```
Comment (patriciogonzalezvivo) [inline 11/README.md:205]: good catch. thanks

File: 15/README.md [hunk 6]
```diff
@@ -58,7 +58,7 @@ You may be thinking that this is unnecessarily complicated... and you are probab
 
 ![Eadweard's Muybridge study of motion](muybridge.jpg)
 
-This level of craftsmanship links back to some of the first optical experiments ever made. For example on games *sprite animations* are very common, and is inevitably to see on it reminiscence to phenakistoscope, zoetrope and praxinoscope.
+This level of craftsmanship links back to some of the first optical experiments ever made. For example on games *sprite animations* are very common, and is inevitably to see on it reminiscence to [phenakistoscope](https://en.wikipedia.org/wiki/Phenakistiscope), [zoetrope](https://en.wikipedia.org/wiki/Zoetrope) and [praxinoscope](https://en.wikipedia.org/wiki/Praxinoscope).
```
Comment (patriciogonzalezvivo) [inline 15/README.md:61]: thank you for the references

File: glossary/textureCube/README.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-## Texture2D
+## TextureCube
```
Comment (patriciogonzalezvivo) [inline glossary/textureCube/README.md:1]: Good catch

File: 13/index.php (hunk 1 above)
Comment (manoylov) [reply 13/index.php]: returned as it was

File: README.md [hunk 8]
```diff
@@ -31,11 +31,11 @@ This is a gentle step-by-step guide through the abstract and complex universe of
     * [Noise](11/)
     * [Cellular noise](12/)
     * [Fractional brownian motion](13/)
-    * Fractals
+    * [Fractals](14/)
```
[the remark this answers is not in the document]
Comment (manoylov) [reply README.md]: returned as it was

File: README-ua.md (hunk 2 above)
Comment (manoylov) [reply README-ua.md]: 👌

File: README-it.md (hunk 3 above)
Comment (manoylov) [reply README-it.md:99]: oops... turned it back

File: 13/index.php (hunk 1 above)
Comment (ManoylovAC) [reply 13/index.php]: returned as it was

File: README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (ManoylovAC) [reply README.md]: 👌 returned as it was

File: README-ua.md (hunk 2 above)
Comment (ManoylovAC) [reply README-ua.md]: Done

File: README-it.md (hunk 3 above)
Comment (ManoylovAC) [reply README-it.md:99]: oops... turned it back

**Comment by patriciogonzalezvivo** [comment]
Thank you!!

**Pull request merged**